src/components/playground/TypographyInput.js

```diff
@@ -145,32 +148,38 @@ function FontFamily(props) {
 
   return (
     <>
-      <TextField
+      <InputLabel id="label" style={{ fontSize: '0.8rem' }}>
+        Font Family
+      </InputLabel>
+      <Select
+        id="standard-select-font-native"
         variant="standard"
         label={label}
         value={value}
         error={error.length > 0}
         onChange={handleChange}
         helperText={error || 'Add Fontfamily from loaded families.'}
+        SelectProps={{
+          native: true,
```

Comment:
`native` is a prop that can directly be used on the `Select` component, but I don't think we need it since it will result in some more errors.